tests/integration/test_interaction.py

```diff
@@ -474,6 +541,15 @@ def test_item_negative_feedback_on_bad_input_with_keyboard(self, action_key):
     def test_final_feedback_and_reset_with_keyboard(self, action_key):
         self.parameterized_final_feedback_and_reset(self.items_map, self.feedback, action_key=action_key)
 
+    def test_alt_text_image(self):
```

Comment:
@dovinmu I believe "displaying" help texts to screen reader should not depend on keyboard vs. mouse control at all, so both normal and keyboard controlled tests should exercise it.

Those new tests are not actually affected by keyboard vs. mouse, except `alt_text_for_occupied_zones`, but even that test only uses mouse vs. keyboard in "arrange" step (arranging pre-conditions), not in "act" step, so to me it does not make sense to actually make it keyboard controlled. However, if you're really want to - take a look at `test_final_feedback_and_reset_with_keyboard` and note that `alt_text_for_occupied_zones` accepts `action_key` parameter too.